package.json

```diff
@@ -50,6 +50,7 @@
     "chai": "^4.1.2",
     "compression-webpack-plugin": "^1.1.6",
     "coveralls": "^3.0.0",
+    "cypress": "^1.4.2",
```

Comment:
can you check if `1.4.2 ` is working? I ran it two days ago and seems there was some breaking change between `1.4.1` and `1.4.2` which breaks `nock` :(

Reply:
Yeah it seems to work: https://travis-ci.org/gr2m/octokit-rest-browser-experimental/jobs/338642214#L504

I don't use nock in the tests, I start a separate server process that I send the requests to, maybe that's why we don't run into the bug?